### PR TITLE
hooks: remove perl-base and adduser

### PR DESCRIPTION
**Pull request by Meulengracht**
This PR drops perl-base and adduser from the base.

Size is reduced 2.2MB

### CODE REVIEW

File: hooks/300-remove-pkgs-late.chroot [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+# following packages depend on adduser (but only during postinst/postrm scripts afaict) 
+# --- wpasupplicant depends on adduser.
+# --- udev depends on adduser.
+# --- openssh-server depends on adduser.
+# --- openssh-client depends on adduser.
+# --- dbus-system-bus-common depends on adduser.
+dpkg --purge --force-depends --force-remove-essential perl-base adduser
+
+# cleanup perl modules left behind by other packages
+find / -type f -name '*.pm' -delete
```
Comment (Meulengracht) [inline hooks/300-remove-pkgs-late.chroot:23]: This is probably not a good idea, I'll remove this for now, and get a proper list of perl-modules left behind in another PR

**Comment by bboozzoo** [comment]
This is what I got:

```
./usr/bin/c_rehash: Perl script text executable
./usr/bin/dh_bash-completion: Perl script text executable
./usr/bin/helpztags: Perl script text executable
./usr/bin/ucfq: Perl script text executable
./usr/lib/ssl/misc/CA.pl: Perl script text executable
./usr/lib/ssl/misc/tsget.pl: Perl script text executable
./usr/sbin/adduser: Perl script text executable
./usr/sbin/deluser: Perl script text executable
./usr/sbin/install-sgmlcatalog: Perl script text executable
./usr/sbin/luksformat: Perl script text executable
./usr/sbin/pam-auth-update: Perl script text executable
./usr/sbin/pam_getenv: Perl script text executable
./usr/sbin/update-catalog: Perl script text executable
./usr/sbin/update-rc.d: Perl script text executable
./usr/share/perl5/Debian/AdduserCommon.pm: Perl5 module source, ASCII text
./usr/share/perl5/Debian/AdduserLogging.pm: Perl5 module source, ASCII text
./usr/share/perl5/Debian/AdduserRetvalues.pm: Perl5 module source, ASCII text
./usr/share/perl5/Debian/Debhelper/Sequence/bash_completion.pm: Perl script text executable
```

**Review by alfonsosanchezbeato — COMMENTED** [review]
It seems that we need to find out if we can safely remove the scripts that @bboozzoo found

**Review by valentindavid — APPROVED** [review]
There does not seem to be any perl script left. Thank you.

**Pull request merged**